toDoList/src/statusUpdate.js

```diff
@@ -0,0 +1,36 @@
+import ToDoList from './toDoList';
+
+const toDoList = new ToDoList();
+
+const checkStatus = (checkBox) => {
+  if (checkBox.classList.contains('true')) {
+    const myCheckBox = checkBox;
+    myCheckBox.checked = true;
+    checkBox.closest('.task').querySelector('.labelText').classList.toggle('lineThrough');
+  }
+};
+
+const changeStatus = (target) => {
+  if (target.classList.contains('checkBox')) {
+    toDoList.toDoTasks = JSON.parse(localStorage.getItem('toDoList'));
+    const parentNode = target.closest('.task');
+    const taskID = parentNode.getAttribute('id') - 1;
```

Comment:
- [ ] **[OPTIONAL]** It is better to use let instead of const for the numeric values.